src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -321,7 +328,9 @@ public sealed class AdoConnector {
                 });
             }
 
-            var systemState = JsonSerializer.Serialize(item.Fields["System.State"]);
+            // the below was causing on_duplicate not to work
+            // var systemState = JsonSerializer.Serialize(item.Fields["System.State"]);
```

Comment:
maybe this was supposed to be Json.Desrialize<string>(item.Fields["System.State"]) ? 

@tevoinea

Reply:
We were debugging it together earlier and there's no serialization required